sdk/communication/Azure.Communication.CallingServer/tests/CallingServerClient/CallingServerClientLiveTests.cs

```diff
@@ -65,7 +67,7 @@ public async Task RunAllRecordingFunctionsScenarioTests(AuthMethod authMethod)
                 Assert.ThrowsAsync<RequestFailedException>(async () => await callingServerClient.GetRecordingStateAsync(recordingId).ConfigureAwait(false));
 
                 // Hang up the Call, there is one call leg in this test case, hangup the call will also delete the call as the result.
-                await SleepIfNotInPlaybackModeAsync().ConfigureAwait(false);
+                await WaitForOperationCompletion().ConfigureAwait(false);
```

Comment:
we dont have to wait for getting call recording state. pl fix in other places in code as well